### ISSUE TITLE
Generated images too dark

### ISSUE THREAD

**Issue by sjtuplayer** [NVlabs/stylegan2-ada-pytorch#152]
I run the code:
          with open('ffhq.pkl', 'rb') as f:
              G = pickle.load(f)['G_ema'].cuda()  # torch.nn.Module
          z = torch.randn([1, G.z_dim]).cuda()    # latent codes
          c = None                                # class labels (not used in this example)
          img = G(z, c)                   # NCHW, float32, dynamic range [-1, +1]
But img do not strictly range from -1 to +1. The minimum number and maximum number may be like -1.2 and 1.1 ,not + and -1.
Moreover, most of the img saved by vutils.save_image are too dark, just like this :
![image](https://user-images.githubusercontent.com/49058490/127817436-4dbba2b1-4537-49cc-8d0a-b75732f49f1e.png)
Could any one tell me why the result is like this. Thanks!

**Comment by pfeducode** [comment]
how do you solve it？

**Comment by pfeducode** [comment]
> > how do you solve it？
> 
> you can just normalie the images. Or when you use save_image, set normalize=True

"I solved it using normalization. First, I scaled the image to [0,1], then multiplied it by 255 and saved it using OpenCV. However, I had to convert the channels to RGB." it works well
